skore/src/skore/sklearn/cross_validate.py

```diff
@@ -35,7 +35,8 @@ def _find_ml_task(
 
     Returns
     -------
-    Literal["classification", "regression", "clustering", "unknown"]
+    Literal["binary-classification", "multiclass-classification",
+    "regression", "clustering", "unknown"]
```

Comment:
I thought about this but since this `MLTask` thing is more of a shortcut type I was reluctant to show it in user-facing docs... but here `_find_ml_task` is not public so maybe that's okay. Will apply